src/Api/WebClient.php

```diff
@@ -188,38 +200,48 @@ public function getAuthorizedHeaderWithContentType(): array
     }
 
     /**
-     * @return mixed
-     *
-     * @throws GuzzleException
+     * @throws InvalidInPostResponseException
+     * @throws ClientExceptionInterface
      */
     public function request(
         string $method,
         string $url,
         array $data = [],
-        bool $returnJson = true
+        bool $returnJson = true,
     ) {
-        $options = [
-            'json' => $data,
-            'headers' => $this->getAuthorizedHeaderWithContentType(),
-        ];
+        $header = $this->getAuthorizedHeaderWithContentType();
 
         try {
-            $result = $this->apiClient->request($method, $url, $options);
-        } catch (ClientException $exception) {
-            /** @var ?ResponseInterface $result */
-            $result = $exception->getResponse();
-
-            throw new ClientException(
-                null !== $result ? (string) $result->getBody() : 'Request failed for url' . $url,
-                $exception->getRequest()
+            $request = $this->requestFactory
+                ->createRequest($method, $url)
+                ->withHeader('Content-Type', $header['Content-Type'])
+                ->withHeader('Authorization', $header['Authorization'])
+                ->withBody($this->streamFactory->createStream(json_encode($data)));
+
+            $result = $this->apiClient->sendRequest($request);
+            $response = json_decode((string) $result->getBody(), true);
+
+            if (201 < $result->getStatusCode()) {
```

Comment:
The max number of 2xx HTTP is 203. To be honest, I don't understand this condition.

Reply:
Fixed:
```
            if (200 !== $result->getStatusCode() && 201 !== $result->getStatusCode()) {
                throw new InvalidInPostResponseException();
            }
```
HHTP 200 - is returned when you select a parcel machine on the map
HTTP 201 - is returned when shipping export action is performed